tests/backup/backup_restore_basic_test.go

```diff
@@ -3712,3 +3713,327 @@ var _ = Describe("{DeleteS3ScheduleAndCreateNfsSchedule}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, s3CloudCredName, s3CloudCredUID, ctx)
 	})
 })
+
+// checkBackupObjectForNonExpectedNS checks if namespaces like kube-system and px namespace
+// is backed up or not
+func checkBackupObjectForNonExpectedNS(ctx context.Context, backupName string) {
```

Comment:
I feel we should take services as input rather than switching clusters internally.

Reply:
How about we return error?

Reply:
The intend of these functions is to reduce code duplication in case of the specific case, returning error and again adding dash.verifyfatel will come back to the same problem of redundant code. Yes it would be 3-4 redundant lines then some 10-20 lines but I feel this way it's much more cleaner.